test/distributed/tensor/test_utils.py

```diff
@@ -19,6 +22,80 @@
 c10d_functional = torch.ops.c10d_functional
 
 
+class LocalTest(TestCase):
+    def test_explicit_order_placements(self):
+        # mesh_shape: ShapeType, placements: Sequence[Placement]
+        test_cases = [
+            {
+                "mesh_shape": [2, 2],
+                "placements": [Replicate()],
```

Comment:
hmmm aren't this need to error out? The rank of mesh_shape and placements should be the same

Reply:
hmm. yea, i probably need to change the function to be more careful here. and the test case should be 2 replicates, ill change that and also test the one replicate one throws an error.